src/__tests__/Search/fetchRevisionByID.test.tsx

```diff
@@ -6,6 +6,7 @@ import getTestData from '../utils/fixtures';
 import { renderWithRouter, store } from '../utils/setupTests';
 import { screen } from '../utils/test-utils';
 
+jest.setTimeout(10000); // sets the timeout to 10 seconds
```

Comment:
Why did you set this timeout to 10 seconds?

Reply:
This was a change suggested on the logs...the previous timeout was causing some tests to fail.